service/routes.py

```diff
@@ -377,26 +377,34 @@ class CustomerResource(Resource):
     #------------------------------------------------------------------
     # RETRIEVE AN EXISTING CUSTOMER
     #------------------------------------------------------------------
+    @api.doc('get_customers')
+    @api.response(404, "Customer not found")
+    @api.marshal_with(customer_model)
     def get(self, customer_id):
         """
         Retrieve a single customer
         This endpoint will return a customer based on their id
         """
-        app.logger.info(f"Request information for customer with customer_id: {customer_id}")
+        app.logger.info("Request information for customer with id [%s]", customer_id)
         customer = Customer.find(customer_id)
         if not customer:
-            raise NotFound(f"Customer with id '{customer_id}' was not found.")
+            raise NotFound("Customer with id '{}' was not found.".format(customer_id))
         return customer.serialize(), status.HTTP_200_OK
 
     #------------------------------------------------------------------
     # UPDATE AN EXISTING CUSTOMER
     #------------------------------------------------------------------
+    @api.doc('update_customers')
+    @api.response(404, 'Customer not found')
+    @api.response(400, 'The posted Customer data was not valid')
```

Comment:
I'm wondering if we just want to make sure that the documentation shows similar error messages for different endpoints that can throw the same error code. For example, wondering if we should change Update an existing address' 400 to 'The posted Address data was not valid'

Reply:
Yeah, I think it's better if we can keep consistency. I fine with using posted/supplied. I can change 'The supplied Address data was not valid' to 'The posted Address data was not valid'

Reply:
I guess the 404 label is different for customers and addresses too right now, not sure whether we want to worry about consistency there

Reply:
I think it's fine for the address to have a more specific explanations, and i think we don't need to worry about the consistency there.